app/Models/Campaign.php

```diff
@@ -326,13 +326,17 @@ public function toSearchableArray()
             $array[$date] = optional($this->{$date})->timestamp;
         }
 
+        // Add boolean attribute for filtering website campaigns.
+        $array['has_website'] = !is_null($this->contentful_campaign_id);
```

Comment:
so this is basically saying, if the `contentful_campaign_id` is not null set `has_website` to true? just to make sure i'm understanding this. haven't looked through php in a minute 😅

Reply:
I'm really grateful you asked this because I second-guessed the double negative but left it in there. You _are_ correct as to the logic, but I updated this to use an [`isset`](https://www.w3schools.com/php/func_var_isset.asp) instead to check that the `contentful_id` has an assigned value and is _not_ `null` in https://github.com/DoSomething/rogue/pull/1146/commits/a7a6501fde8fcae2cdb1aa904bbac0c4630c6b32 for readability!